src/main/java/javi/sonarqube/ComplexMethod.java

```diff
@@ -0,0 +1,33 @@
+package javi.sonarqube;
+
+import java.util.logging.*;
+
+public class ComplexMethod {
+
+    public String intToEnglishValue(int i) throws Exception {
+        System.out.println("Hello...");
+    	if (i == 1) {
+            return "One";
+        }
+        if (i == 2) {
+            return "Two";
+        }
+        if (i == 3) {
+            return "Three";
+        }
+        if (i == 4) {
+            return "Four";
+        }
+        if (i == 5) {
+            return "Five";
+        }
+        if (i == 6) {
+            return "Six";
+        }
+        if (i > 6) {
+            throw new Exception();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS00112)